From 643f50d50871d6659c6bd40da5eac9552632e3bd Mon Sep 17 00:00:00 2001
From: "Ralf W. Grosse-Kunstleve" <rgrossekunst@nvidia.com>
Date: Thu, 19 Jun 2025 09:05:50 -0700
Subject: [PATCH 1/5] Update pybind11 submodule to latest master

---
 third_party/pybind11 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/third_party/pybind11 b/third_party/pybind11
index a2e59f0e706..03d8f48750b 160000
--- a/third_party/pybind11
+++ b/third_party/pybind11
@@ -1 +1 @@
-Subproject commit a2e59f0e7065404b44dfe92a28aca47ba1378dc4
+Subproject commit 03d8f48750ba4486a2c9aeff82e9702109db5cb3
-- 
2.43.0

